src/pages/components/tabs-extended.mdx

```diff
@@ -16,107 +16,124 @@ import ResourceLinks from 'components/ResourceLinks';
 
-<Caption>Example of the tabs extended, on desktop</Caption>
+<Caption>Example of the tabs extended on desktop</Caption>
 
 </Column>
 </Row>
 
-On mobile, Tabs extended switches to an accordion for easier browsing.
+Like tabs, tabs extended will expand a given item horizontally to display the whole label without truncating. Keep this in mind as you write selection items for tabs extended, as overly long labels can push other options out of the initial view. Also consider translation lengthening the label.
 
-<Row>
-<Column colMd={6} colLg={4}>
+## Tabs extended
 
-![Image of Tabs extended component on `sm`](../../images/component/tabs-extended/components--tabs-extended-3.png)
+Tabs extended can nest other components inside its content area to display them based on the user's selection. We recommend nesting similar components in sequence, rather than varying the types of components displayed in a given tabs experience.
 
-<Caption>Example of the tabs extended, on mobile</Caption>
+<ResourceLinks name="Tabs extended" type="ui" multiComponent />
 
-</Column>
-</Row>
+### Content guidance for Tabs extended
 
-### Modifiers
+| Element                                                                                                                | Content type | Required | Instances | Character limit <br/>(English / translated) | Notes                                                                                         |
+| ---------------------------------------------------------------------------------------------------------------------- | ------------ | -------- | --------- | ------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| Child container                                                                                                        | Component    | Yes      | Min 2     | –                                           | Container areas that child components and other content types can be passed into (1 per tab). |
+| <a href="https://www.carbondesignsystem.com/components/tabs/usage/" target="_blank" rel="noopener noreferrer">Tabs</a> | Component    | Yes      | 2–6       | –                                           |                                                                                               |
+| Tab item                                                                                                               | Text         | Yes      | –         | 25 / 35                                     |                                                                                               |
 
-#### Orientation
+For more information, see the [character count standards](https://www.ibm.com/standards/carbon/guidelines/content#character-count-standards).
+
+## Tabs extended with media
```

Comment:
Let's double check with the devs.. I read through this whole page then remembered we maybe deprecated tabs extended with media?